lib/game.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+require_relative 'ui.rb'
+
+class Game
+  attr_accessor :continue
+  def initialize(board, player1, player2, ui)
+    @new_board = board
+    @player1 = player1
+    @player2 = player2
+    @ui = ui
+    @ui.show_board(@new_board)
+    play
+  end
+
+  def play
+    @player1.assign_sign(@ui)
+    @player2.sign = @player1.sign == 'X' ? 'O' : 'X'
+    while @new_board.check_board
+      cell = @ui.choose_num(@new_board)
+      @player1.make_move(@new_board, cell.to_i)
+      @ui.show_board(@new_board)
+      is_winner = check_game
+      if is_winner == true
+        self.continue = @ui.player1_msg
+        return
+      elsif is_winner == false && !@new_board.check_board
+        self.continue = @ui.draw_msg
+        return
+      end
+      cell = @ui.choose_num(@new_board)
+      @player2.make_move(@new_board, cell.to_i)
+      @ui.show_board(@new_board)
+      is_winner = check_game
+
+      if is_winner == true
+        self.continue = @ui.player2_msg
+        return
+      elsif is_winner == false && !@new_board.check_board
+        self.continue = @ui.draw_msg
+        return
+      end
+
+    end
+  end
+
+  def check_game
```

Comment:
This method is simply returning the return of another method. Maybe we can just invoke `winner` in the place where we are currently invoking `check_game`.